meta-lmp-base/recipes-support/luks-reencryption/luks-reencryption/luks-reencryption

```diff
@@ -13,9 +13,13 @@ DEVICE="/dev/${DEVICE}"
 # Avoid using the PIN for OP-TEE supported PKCS#11 user authentication
 export CKTEEC_LOGIN_TYPE=user
 
+LUKS_HEADER_BACKUP=luks.bin
+
 if cryptsetup luksDump ${DEVICE} | grep -q "online-reencrypt"; then
 	# Preemptively check if the volume needs to be repaired
 	yes "YES" | cryptsetup -v repair ${DEVICE}
 	# Resume reencryption
 	cryptsetup reencrypt --resume-only ${DEVICE}
+	# Backup header file
+	cryptsetup luksHeaderBackup ${DEVICE} --header-backup-file /boot/${LUKS_HEADER_BACKUP}
```

Comment:
Should we check if there is store available here?

Reply:
I think we just need to check if the backup creation fails and raise a warning.